Functions.Templates/Templates/KafkaTrigger-CSharp/KafkaTriggerCSharp.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
```

Comment:
Can this file be split into 2 individual templates?

Reply:
Sure. I removed the KafkaOutput I'll add the KafkaOutput with following pull requests in small sized ones.

Reply:
Done. I'll separate the KafkaOutput. I'll send via separate small prs.